src/Components/Calculator.js

```diff
@@ -1,42 +1,84 @@
-import React from "react";
+import React, { useState } from "react";
 import "./Calculator.css";
 
-class Calculator extends React.Component {
+const Calculator = () => {
+    let [calc, setCalc] = useState("");
+    let [result, setResult] = useState("");
+
+    const ops = ['/', '*', '+', '-', '%', '.'];
+
+    const evaluation = (value) => {
+      if(eval(calc + value).toString() === 'Infinity'){
+        setResult("Can't divide by 0.");
+      }
+      else if (eval(calc + value).toString() === 'NaN'){
+        setResult("Can't find modulo as can't divide by 0.");
+      }
+      else{
+        setResult(eval(calc + value).toString());
+      }
+    }
+
+    const updateCalc = value => {
+      if(ops.includes(value) && calc === '' || ops.includes(value) && ops.includes(calc.slice(-1))){
+        return;
+      }
+
+      
+      if (!ops.includes(value)){
+        evaluation(value);
+      }
+
+      setCalc(calc + value);
+  
+    }
+
+    const calculate = () => {
+      setCalc('');
+      evaluation();
+      // setResult(eval(calc).toString());
```

Comment:
- [ ] please avoid leaving in commented out code, it will make your code base look unorganized and harder to read.